wordpress-mobile/WordPress-iOS
# AztecPostViewController: Updating title position upon mode switch

### Details:
In this PR we're patching a glitch in which the Title field would become invisible.

Fixes #8033
 
### To test:
1. Launch Aztec
2. Switch to HTML Mode
3. Paste the snippet pasted below
4. Switch to Visual Mode

Verify that the Title Field remains visible.

Needs review: @diegoreymendez 


```
We hiked along the Pacific, in the town of <a href="http://aaaaaa/" target="_blank">Mendocino</a>. <i>Stunning views! </i><img class="size-large wp-image-80" src="https://thenomadicwordsmith.files.wordpress.com/2014/10/img_8934.jpg?w=700" alt="" width="700" height="505">
<p style="text-align: left">It's a <b>beautiful</b> region in Northern California. We hope to visit again soon!</p>
<p style="text-align: left">Along this coastal trail, you'll see:</p>

<ul style="text-align: left">
 	<li>lots of sea caves</li>
 	<li>rock formations</li>
 	<li>marine life</li>
</ul>
```
